extensions/SemanticForms/specials/SF_UploadWindow.php

```diff
@@ -327,7 +327,7 @@ protected function processUpload() {
 		if ( !$status->isOK() )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)